contracts/l2/L2DAIWormholeBridge.sol

```diff
@@ -80,6 +82,19 @@ contract L2DAIWormholeBridge is OVM_CrossDomainEnabled {
     emit Closed();
   }
 
+  function file(
+    bytes32 what,
+    bytes32 domain,
+    uint256 data
+  ) external auth {
+    if (what == "validDomains") {
+      validDomains[domain] = data;
```

Comment:
Should we maybe force `data` to be either 0 or 1 using something like `validDomains[domain] = data & 1;` ?

Reply:
We don't normally do that check in other places. I would be okay with adding a revert for that, but definitely don't want to coerce the value silently.

Reply:
Or alternatively, use `require(validDomains[targetDomain] != 0, "L2DAIWormholeBridge/invalid-domain");` instead of `require(validDomains[targetDomain] == 1, "L2DAIWormholeBridge/invalid-domain");` on L108.

Reply:
Actually, I'm mixing things up. Thinking of the `live` non-bool optimization, but that doesn't allow freeform setting the value to any `uint256`. I think I'll just restrict the value in the `file` as there is gas savings using the `uint256` read vs the `bool`.

Reply:
We could just switch to bool as well as the gas difference is not that large.

Reply:
I'd enforce value in file.